consent/__init__.py

```diff
@@ -0,0 +1,3 @@
+"""
+edX Enterprise's Consent application.
```

Comment:
Yeah the docstrings definitely need work, I just didn't want to spend too much time on them just yet, so thank you for this example; it looks good enough to be straight dropped right into it. I'll edit the rest to reflect something similar in a shorter fashion where appropriate.